### ISSUE TITLE
Tests failing on Django 1.9

### ISSUE THREAD

**Issue by hugorodgerbrown** [yunojuno/django-basecrm#2]
Running the 1.9 tests using `tox -e django19_py27`, there's been an api change in Django which breaks the tests:

```
======================================================================
ERROR: Failure: ImportError (No module named loading)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/src/github.com/yunojuno/django-basecrm/.tox/django19_py27/lib/python2.7/site-packages/nose/loader.py", line 420, in loadTestsFromName
    addr.filename, addr.module)
  File "/src/github.com/yunojuno/django-basecrm/.tox/django19_py27/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/src/github.com/yunojuno/django-basecrm/.tox/django19_py27/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/src/github.com/yunojuno/django-basecrm/django_basecrm/tests.py", line 9, in <module>
    from . import apps, exceptions, helpers, serializers, utils
  File "/src/github.com/yunojuno/django-basecrm/django_basecrm/serializers.py", line 6, in <module>
    from django.db.models.loading import get_model
ImportError: No module named loading
```

**Comment by hugorodgerbrown** [comment]
The 1.9 tests have been removed from the default tox setup, so that CI builds don't break.

**Comment by hugorodgerbrown** [comment]
Fixed by #5
